api/pkg/parser/catalog.go

```diff
@@ -71,7 +72,7 @@ func (c *CatalogParser) Parse() ([]Resource, Result) {
 	resources := []Resource{}
 	result := Result{}
 
-	kinds := []string{"Task", "Pipeline"}
+	kinds := resource.SupportedKinds()
```

Comment:
I think we should keep this method in `parser` package, outside the services, so that services don't have to depend on each other.

Reply:
/parser/resource/kind.go -> `resource.SupportedKinds()` ?

Reply:
/pkg/resource/kind.go seems better to me than keeping in parser. wdyt?